export const createCheckoutSession = async (req, res) => {
    try {

    } catch (error) {

    }
}

export const checkoutSuccess = async (req, res) => {
    try {

    } catch (error) {

    }
}